src/utils/api/simApi.ts

```diff
@@ -0,0 +1,111 @@
+import { type RequestGame } from '../../types/apiData'
+import {
+  type PostGamesResponse,
+  type PostGamesReturnValue,
+  type GetGamesResponse,
+  type GetGamesReturnValue,
+  type DeleteGameResponse,
+  type DeleteGameReturnValue,
+} from './returnValues'
+import {
+  AuthorizationError,
+  InternalServerError,
+  NotFoundError,
+  UnprocessableEntityError,
+} from './apiErrors'
+
+const BASE_URI = import.meta.env.PROD
+  ? 'https://sim-api.danascheider.com'
+  : import.meta.env.MODE === 'test'
```

Comment:
MSW was having trouble intercepting the API calls and it turned out this was why.